javatests/build/bazel/tests/integration/CommandTest.java

```diff
@@ -0,0 +1,27 @@
+package build.bazel.tests.integration;
+
+import org.junit.Test;
+
+import java.io.IOException;
+import java.util.Collections;
+import java.util.Map;
+
+import static org.hamcrest.core.IsCollectionContaining.hasItem;
+
+public class CommandTest {
+
+    @Test
+    public void runWithEnvironment() throws IOException, InterruptedException {
```

Comment:
How much value does the test give us over the IT one?

Reply:
Not a lot... Do you think it should be removed?

Reply:
yes